src/fragments/guides/storage/transfer-accelaration.mdx

```diff
@@ -0,0 +1,43 @@
+You can enable Transfer Acceleration for fast and secure transfer of files over long distances between your end user device and the S3 bucket. You can override the storage resource for this configuration and then leverage the `useAccelerateEndpoint` parameter to use the accelerated S3 endpoint.
+
+## Override storage resource
+
+Start by overriding your storage resources to enable Transfer Acceleration on your S3 bucket.
+
+```sh
+$ amplify override storage
+✅ Successfully generated "override.ts" folder at <project>/amplify/backend/storage/accelarated-bucket
+✔ Do you want to edit override.ts file now? (Y/n) · yes
+Edit the file in your editor: <project>/amplify/backend/storage/accelarated-bucket/override.ts
+```
+
+In the generated `override.ts` file use the following CDK snippet to enable transfer accelaration. 
+
+```typescript
+// amplify/backend/storage/accelarated-bucket/override.ts
+import { AmplifyS3ResourceTemplate } from '@aws-amplify/cli-extensibility-helper';
+
+export function override(resources: AmplifyS3ResourceTemplate) {
+    resources.s3Bucket.accelerateConfiguration = {
+        accelerationStatus: 'Enabled'
+    }
+}
+```
+
+Next, deploy this storage resource:
+
+```sh
+amplify push
+```
+
+
+
+## Use below snippet for leveraging accelarated endpoint
+
+While using the Storage.put method, we use the `useAccelerateEndpoint` parameter to use the accelarated S3 endpoint.
```

Comment:
writing nit: two usages of 'use' in the same sentence doesn't read very well.

Reply:
addressed wording for readability